<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>enter_recipe</class>
 <widget class="QWidget" name="enter_recipe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>嘉然有了新菜谱！( •̀ ω •́ )✧</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main_jiaran.qrc">
    <normaloff>:/1/3.JPG</normaloff>:/1/3.JPG</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="enter_recipe_back">
   <property name="geometry">
    <rect>
     <x>1470</x>
     <y>0</y>
     <width>130</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_name">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>135</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>菜名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择想要增加的菜~</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>地点</string>
   </property>
  </widget>
  <widget class="QComboBox" name="input_location">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>215</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="input_location2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>215</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>价格</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="input_price">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>295</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>（建议取值在5~25元）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>甜度</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="input_sweet">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>375</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>（建议取值在1~3）</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="input_spicy">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>455</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>460</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>辣度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>460</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>（建议取值在1~3）</string>
   </property>
  </widget>
  <widget class="QComboBox" name="input_kind">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>535</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>540</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>种类</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="input_score">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>615</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="input_waiting">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>695</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>620</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>分数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>620</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>（建议取值在1~5）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>700</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>（建议取值在1~5）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>700</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>等待时长</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>670</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文彩云</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 85, 255);</string>
   </property>
   <property name="text">
    <string>录入菜品！</string>
   </property>
   <property name="icon">
    <iconset resource="main_jiaran.qrc">
     <normaloff>:/1/buttom1.png</normaloff>:/1/buttom1.png</iconset>
   </property>
  </widget>
  <widget class="QComboBox" name="input_time">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>215</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>220</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>时间</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="main_jiaran.qrc"/>
 </resources>
 <connections/>
</ui>
